### PR TITLE
Alerting docs: separates doc on create alerts from panels

### PR DESCRIPTION
**Pull request by brendamuir**
separates out create alerts from panels from configure alert rules and adds some more info around viewing.

### CODE REVIEW

File: docs/sources/alerting/alerting-rules/create-alerts-panels.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - cloud
+    - enterprise
+    - oss
+title: Create alert rules from panels
+weight: 400
+---
+
+## Create alert rules from panels
+
+Create alert rules from any panel type. By doing so, you can reuse the queries in the panel and create alert rules based on them.
```
Comment (tonypowa) [inline docs/sources/alerting/alerting-rules/create-alerts-panels.md:22]: actually, the only panels that support alerts ATM are the time-series and alert list panels 😶

Comment (brendamuir) [reply docs/sources/alerting/alerting-rules/create-alerts-panels.md:22]: so say Create alert rules from any time series panel then?

Comment (brendamuir) [reply docs/sources/alerting/alerting-rules/create-alerts-panels.md:22]: I also see your note and play - that will also need to come out into the new doc, right?

**Review by tonypowa — APPROVED** [review]
LGTM!

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md [hunk 2]
```diff
@@ -145,9 +145,7 @@ Annotations add metadata to provide more information on the alert in your alert
    Links alerts to panels in a dashboard.
 
    {{% admonition type="note" %}}
-   At the moment, alerts are only supported in the [time series](ref:time-series) and [alert list](ref:alert-list) visualizations.
+   At the moment, alert rules are only supported in [time series](ref:time-series) and [alert list](ref:alert-list) visualizations.
```
Comment (gillesdemey) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:148]: 👌

File: docs/sources/alerting/alerting-rules/create-alerts-panels.md
Comment (gillesdemey) [inline docs/sources/alerting/alerting-rules/create-alerts-panels.md:1]: General thought on this topic; we should probably include some sort of caveat stating that any changes to the panel isn't reflected on the linked alert rules – the queries would have to be updated in both places once a rule is created from a panel.

**Review by gillesdemey — APPROVED** [review]
LGTM; just a single comment

**Pull request merged**